.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": true
+        }
+    ]
+}
```

Comment:
@jbrown-xentity had mentioned a while ago about adding a debugger configuration to the project. I think it doesn't make sense to add it through this branch though. I'm also okay with omitting it entirely, adding it to gitignore, as people may have different debugging preferences.

Reply:
Oh, this seems like a vscode specific debugger though?

Reply:
yeah it's a vscode debugger. Is that a topic for discussion amongst the team?

Reply:
I removed the folder and added it to gitignore but can easily add it back if need be.

Reply:
I think the hope was to document the vs code "happy path", but not necessarily put it in code (or save the configuration files). I see this ideally as a wiki page that the readme links to. We don't all have to use the same tools if people don't like vs code, but having a working proven use case in something like vs code allows others to follow documentation in their preferred ide to get a similarly working debugger.

Reply:
yeah okay that makes sense ( providing users with a practical use case of debugging harvester 2.0 functionality ). It could also be helpful to include an example of debugging tests.